tools/src/test/scala/caliban/tools/CodegenSpec.scala

```diff
@@ -0,0 +1,89 @@
+package caliban.tools
+
+import Codegen.getPackageAndObjectName
+import zio.test.Assertion._
+import zio.test._
+import zio.test.environment.TestEnvironment
+
+object CodegenSpec extends DefaultRunnableSpec {
+
+  override def spec: ZSpec[TestEnvironment, Any] =
+    suite("CodegenSpec")(
+      test("Package name is empty or default if path doesn't follow correct format") {
+        assert(packageAndObject("a/b/c", None, None))(
+          equalTo(None -> "Client")
+        )
```

Comment:
You need to join the `assert` with `&&`, otherwise only the last one will be executed (it's `assert` from zio-test, not the regular scala assert, it's a pure function that returns a test result).

Reply:
👍